doc/intl/de_GER/README.md

```diff
@@ -1,18 +1,11 @@
 <h1 align="center">
-<img src="https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" alt="Dogecoin" width="300"/>
+<img src="https://raw.githubusercontent.com/dogecoin/dogecoin/master/share/pixmaps/dogecoin256.svg" alt="Dogecoin" width="256"/>
```

Comment:
I don't really agree here, because on the original English version the logo is 256px (I think it depends on the size of the source logo). https://github.com/dogecoin/dogecoin/blob/master/README.md?plain=1

Reply:
the .svg deffo is 256px * 256px unless the sourcefile (dogecoin256.svg) was altered or changed.
Imho that aligns just perfectly with PR 3643 (https://github.com/dogecoin/dogecoin/pull/3643), when the logo is not being derived from an external host (tumblr) anymore but instead sourced from the github repo.. <- exactly what you did. 💯👌

To me, this change of yours is just perfectly executed!..?
pls elaborate if I don't see your point / reason to "not really" agree.

![Screenshot 2024-12-19 at 11 22 55 PM](https://github.com/user-attachments/assets/9fa4c738-4b96-49c5-a772-17ca0e16c814)